HashRegistrarSimplified.sol

```diff
@@ -140,7 +140,7 @@ contract Registrar {
         } else {
             if(entry.highestBid == 0) {
                 return Mode.Open;
-            } else if(entry.deed == Deed(0)) {
+            } else if(minLength(now) == 6 && entry.deed == Deed(0)) {
```

Comment:
This means that any names invalidated during the first 6 months will still show up as owned until the 6 month point, at which point they all become forbidden at once - it still won't allow anyone to reregister those names.